### PR TITLE
Examples: Clean up

### PR DESCRIPTION
**Pull request by Mugen87**
More `Geometry` removal.

### CODE REVIEW

**Comment by Mugen87** [comment]
https://rawgit.com/Mugen87/three.js/dev11/examples/webgl_camera.html
https://rawgit.com/Mugen87/three.js/dev11/examples/webgl_lines_dashed.html
https://rawgit.com/Mugen87/three.js/dev11/examples/webgl_physics_cloth.html
https://rawgit.com/Mugen87/three.js/dev11/examples/webgl_physics_rope.html
https://rawgit.com/Mugen87/three.js/dev11/examples/webgl_physics_terrain.html

File: examples/webgl_physics_rope.html [hunk 1]
```diff
@@ -35,7 +35,7 @@
 	<script src="../build/three.js"></script>
 	<script src="js/libs/ammo.js"></script>
 	<script src="js/controls/OrbitControls.js"></script>
-        <script src="js/Detector.js"></script>
+  <script src="js/Detector.js"></script>
```
Comment (mrdoob) [inline examples/webgl_physics_rope.html]: Minor O:)

Comment (Mugen87) [reply examples/webgl_physics_rope.html]: Ups, there were still some soft tabs in that file.

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**